bands/templates/band_admin/band_info.html

```diff
@@ -11,183 +11,184 @@
 
 <h2>Band Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
 
+<br/> <br/>
 <form class="form-horizontal" role="form" method="post" action="band_info">
     {% csrf_token %}
     <input type="hidden" name="id" value="{{ band.id }}" />
 
-    {% if band.panel_name %}
-        <div class="row">
-            <label class="col-sm-2 control-label">Panel Name:</label>
-            <div class="col-sm-6">
-                {{ band.panel_name }}
-            </div>
+    {% if band.panel.name %}
```

Comment:
hey, as a quick test I ran the following code:

```
{% if c.fake.fake.fake.obviously_bad %}
    YUP
{% else %}
   NOPE
{% endif %}
```

the output was 'NOPE', so it does appear to consider None values as the equivalent of False, so this code should work as intended.  Also outputs nothing if you leave off the 'elif' here too.  Also tested with ```c.DEV_BOX.obviously_wrong``` and same effect.